### ISSUE TITLE
terraform init failes as binary isn't executable

### ISSUE THREAD

**Issue by ledzepp4eva** [ukfast/terraform-provider-safedns#4]
### What is the issue?

After you initilise the provider and try to run it you get the error:
`failed to instantiate provider "registry.terraform.io/ukfast/safedns" to obtain schema: fork/exec .terraform/providers/registry.terraform.io/ukfast/safedns/1.1.2/linux_amd64/terraform-provider-safedns_v1.1.2: permission denied`

### Steps To Reproduce

```hcl
terraform {
  required_version = ">= 0.12"

  required_providers {
    cloudflare = {
      source  = "cloudflare/cloudflare"
      version = "~> 3.0"
    }
    safedns = {
      source  = "ukfast/safedns"
      version = "1.1.2"
    }
  }
}
provider "safedns" {
  api_key = var.safedns_api_token
}

provider "cloudflare" {
  email      = var.cloudflare_email
  account_id = var.cloudflare_account_id
  api_token  = var.cloudflare_api_token
}
```

`terraform init`
`terraform plan`

### Expected behaviour

I'd expect the file `.terraform/providers/registry.terraform.io/ukfast/safedns/1.1.2/linux_amd64/terraform-provider-safedns_v1.1.2` to have the executable bit set.

**Comment by ledzepp4eva** [comment]
I'm guessing the fix would be somewhere in here: https://github.com/ukfast/terraform-provider-safedns/blob/master/.goreleaser.yml

**Comment by 0x4c6565** [comment]
As discussed - this has been resolved. Re-released `v1.1.2` + tagged `v1.1.3`
